src/generators/gen.ts

```diff
@@ -45,8 +74,22 @@ export type BaseRootContext<
   language: string;
   rudderTyperVersion: string;
   trackingPlanURL: string;
-  tracks: (T & BaseTrackCallContext<P>)[];
   objects: (O & BaseObjectContext<P>)[];
+  tracks: (T & BaseTrackCallContext<P>)[];
+  page: (T & BaseTrackCallContext<P>)[];
+  screen: (T & BaseTrackCallContext<P>)[];
```

Comment:
Screen API does not apply to JavaScript SDK.

Reply:
We are not using this for the JS SDK, but since the same generator is written for Node, the template for Node will include the relevant data, whereas it will not be included for the JS SDK inside handlebars.

Reply:
Ok. There is a reference to `screen` in `index.hbs` file we'd want to remove.